### PR TITLE
add more tests

### PR DESCRIPTION
**Pull request by SimonDanisch**

### CODE REVIEW

File: test/runtests.jl [hunk 1]
```diff
@@ -3,6 +3,11 @@ using MakieGallery
 using Test
 using GLMakie
 using StaticArrays, GeometryBasics
+using Observables
+using GeometryBasics: Pyramid
+using PlotUtils
+using MeshIO, FileIO
+using AbstractPlotting: Pixel
```
Comment (asinghvi17) [inline test/runtests.jl:10]: do we not export Pixel?

File: test/runtests.jl [hunk 2]
```diff
@@ -3,6 +3,11 @@ using MakieGallery
 using Test
 using GLMakie
 using StaticArrays, GeometryBasics
+using Observables
```
Comment (asinghvi17) [inline test/runtests.jl:6]: We should reexport all these utility packages (Colors, Observables)

File: test/reference_image_tests/simple.jl [hunk 3]
```diff
@@ -0,0 +1,208 @@
+using MakieGallery: @substep
+
+@block SimonDanisch ["tests"] begin
+    ## Scatter
+    @cell begin
```
Comment (asinghvi17) [inline test/reference_image_tests/simple.jl:5]: would multiple cells be better here?  all of these in one cell makes it kind of hard to debug.

Comment (asinghvi17) [reply test/reference_image_tests/simple.jl:5]: Also, we do need to separate 2d examples from 3d ones (for CairoMakie)

File: test/runtests.jl (hunk 2 above)
Comment (jkrumbiegel) [reply test/runtests.jl:6]: yes, `onany` and `Gray` are two missing exports that always annoy me :)

File: test/reference_image_tests/simple.jl (hunk 3 above)
Comment (SimonDanisch) [reply test/reference_image_tests/simple.jl:5]: Yeah I need to figure out how to do this... I had to put them in one cell, to have theglobally shared functions and data work!

Comment (asinghvi17) [reply test/reference_image_tests/simple.jl:5]: You could also put toplevel code in, that works now

**Pull request merged**